boards/risc-v/mpfs/icicle/scripts/ld-envm-opensbi.script

```diff
@@ -59,7 +59,7 @@ SECTIONS
 
     .text : {
         _stext = ABSOLUTE(.);
-        *(.vectors)
+        *mpfs_head.o
```

Comment:
Why do we need `*` here?

Reply:
In this case it is actually not needed, you are right. I can remove it.